src/arguments/api_test.py

```diff
@@ -84,6 +116,20 @@ def get_request(cls):
             print(cls.invalid_schema_message)
         except Exception as exception_obj:
             print(exception_obj)
+    #Make a POST request
+    @classmethod
+    def post_request(cls):
+        request_data = cls.fetch_input_url()
+        data = cls.fetch_payload_data()
+        try:
+            response= requests.post(url = request_data["request_url"], headers= request_data["request_headers"], data = data)
```

Comment:
Add a space before = (equals)

Reply:
everything can be done, using black formatter I guess, shall we try adding it

Reply:
yes, you can use the black formatter locally on your system before pushing it 😃